CHARTER.md

```diff
@@ -0,0 +1,104 @@
+# MapLibre Charter
+
+## Definitions
+
+*MapLibre Organization:* An umbrella organization to develop, maintain and promote MapLibre software projects
+
+*MapLibre Projects:* multiple repositories, each being some part of the map visualization stack from the raw data to the map visualization and interaction.
+
+*MapLibre:* As a single word, “MapLibre” is a brand name used by the front-end map rendering MapLibre projects (i.e. code libraries).
+
+*MapLibre Community:* People interested in the MapLibre Projects. Anyone can be part of the MapLibre Community and there are no special requirements except to follow community code of conduct. For example, the Community members … :
+
+ * evangelize the MapLibre Project
+ * educate new users of project’s strengths and weaknesses
+ * provide moral support (a ‘thank you’ goes a long way)
+ * support the MapLibre Organization financially
+ * improve code and fix bugs
+ * help with documentation
+ * give advice to others
+
+*Technical Steering Committee:* An informal group of people who meet on a monthly basis and discuss technical topics related to software development. Anyone is free to join the Technical Steering Committee.
+
+*Voting Member:* A person who can elect the Governing Board and vote on changes to the MapLibre Charter.
+
+*Governing Board:* The elected representatives of the MapLibre Community who help shape the Project's overall direction and who decide over the Project's finances.
+
+*Third-party Organization:* A company or organization that uses MapLibre software for commercial or non-commercial purposes.
+
+## MapLibre Organization Goals
+The MapLibre Organization develops free open-source mapping libraries and tools for others to build applications and services.
+
+The primary goal of the MapLibre Organization is to build a friendly and innovative open-source community around the unaffiliated MapLibre Projects.
+
+The MapLibre Organization aims to encourage code development. We prefer to empower our users to build end-user applications, design maps, run map hosting services or distribute data rather than making it part of the MapLibre Projects.
+
+## Governance Bodies
+The MapLibre Organization has three Governance Bodies which support and represent the broader MapLibre Community:
+
+ * Technical Steering Committee
+ * Governing Board
+ * Voting Members
+
+### Technical Steering Committee
+The Technical Steering Committee discusses technical topics that affect multiple MapLibre projects, coordinates ongoing work, and decides if proposed changes with a major impact should be accepted or not.
```

Comment:
I may be conflating the TSC with the monthly meeting, but I'm not sure this is where final decisions on accepting proposed changes should live, since the TSC may not represent relevant stakeholders fully. The clarification below about individual projects forming their own steering committees may negate my concern though. Curious what others think here.

Reply:
IMO: TSC has the authority to take informal, generally binding votes, but the governing board has the authority to overrule these votes.

That may be overly pragmatic, but I'm comfortable with that for now.

Reply:
The Technical Steering Committee meets on a monthly basis and we want it to be open to everyone. If you come regularly to these meetings, you will be part of the Technical Steering Committee. We will only have an election for the Governing Board and if there is a dispute about a technical thing, it is the Governing Board who will decided what to do.

Reply:
So far the Technical Steering Committee meetings did not lead to decisions which would have gone against the will of somebody who my coincidence did not attend. But of course this might happen. As @klokan said recently, in an open-source project the people who contribute a lot and are around are the ones who shape the project...

Reply:
Closing this for now, feel free to reopen, @mojodna...